crate/operator/main.py

```diff
@@ -382,129 +382,169 @@ async def cluster_create(
         )
 
 
-@kopf.on.update(API_GROUP, "v1", RESOURCE_CRATEDB)
-async def cluster_update(
-    namespace: str,
-    name: str,
-    body: kopf.Body,
-    spec: kopf.Spec,
-    diff: kopf.Diff,
-    old: kopf.Body,
-    logger: logging.Logger,
-    **kwargs,
+@kopf.on.update(API_GROUP, "v1", RESOURCE_CRATEDB, id="update", backoff=10)
```

Comment:
As discussed in the daily coffee, I'm not sure if the `id` and the `backoff` are needed.